acceptance/lib/helper.rb

```diff
@@ -290,8 +290,7 @@ def https_request(url, request_method, cert, key, body = nil)
   end
 
   def hup_server(host = master, timeout = 30)
-    pidfile = on(host, 'puppet master --configprint rundir').stdout.chomp + '/puppetserver' 
-    pid = on(host, "cat #{pidfile}").stdout.chomp
+    pid = on(host, "pgrep -f #{options['puppetservice']}").stdout.chomp
```

Comment:
@rlinehan didn't the hyphen changes get backed out? shouldn't it be puppet-server if you want to find the jar? (i think puppetserver is in the path to the jar, but isn't the jarfile itself)